pages/blog/houdini-and-kitql.mdx

```diff
@@ -0,0 +1,218 @@
+import withArticle from '../../ui/blog/article'
+
+export const meta = {
+  title: 'Bringing the best GraphQL experience to Svelte',
+  tags: ['graphql', 'svelte'],
+  authors: ['jycouet', 'AlecAivazis'],
+  date: '2022-07-04',
+  description:
+    "It's time to unveil a massive effort by the KitQL and Houdini maintainers to join forces!",
+  image: '/blog-assets/houdini-and-kitql/2022 05 25 - KitQL & Houdini.gif',
+  thumbnail: '/blog-assets/houdini-and-kitql/thumbnail.png'
+}
+
+export default withArticle(meta)
+
+## A Tale of Two Projects
+
+[SvelteKit](https://kit.svelte.dev/) is taking the web dev community by storm by providing an excellent solution
+for server-side rendered web applications and simplifying the creation of "WebApp"s
+... whatever that means 😅. While most of the common libraries for GraphQL apps do
+have svelte bindings, none of them really _spark joy_. It was hard to avoid this feeling
+that people were trying to smash a React-shaped wedge into a Svelte-shaped hole.
+Independently, [Alec](https://github.com/AlecAivazis) and [Jean-Yves](https://twitter.com/jycouet)
```

Comment:
I think that you can replace "Jean-Yves" with "I" since you are the author of the article 👀